.changeset/lemon-foxes-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+Add Scaffolder action to get entity by entity reference from catalog
```

Comment:
Think it would be nice to mention the ID of the added action here.

Reply:
Yep, will add it once I change it